DatabaseDesign/Database_design_SQL

```diff
@@ -0,0 +1,80 @@
+SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0;
+SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0;
+SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='TRADITIONAL,ALLOW_INVALID_DATES';
+
+CREATE SCHEMA IF NOT EXISTS `mydb` DEFAULT CHARACTER SET utf8 COLLATE utf8_general_ci ;
+USE `mydb` ;
+
+-- -----------------------------------------------------
+-- Table `mydb`.`Salary`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `mydb`.`Salary` (
+  `employee_id` INT NOT NULL,
+  `salary_amount` DOUBLE NOT NULL,
+  `start_date` DATE NOT NULL,
+  `end_date` DATE NOT NULL,
+  `incentive_amount` DOUBLE NULL,
+  `bonus_amount` DOUBLE NULL,
+  `over_time_amount` VARCHAR(45) NULL,
+  UNIQUE INDEX `Employee ID_UNIQUE` (`employee_id` ASC),
+  UNIQUE INDEX `Date_UNIQUE` (`start_date` ASC),
+  UNIQUE INDEX `End_Date_UNIQUE` (`end_date` ASC))
```

Comment:
Can you explain why `start_date` and `end_date` are unique here? I really don't see the point.